builder.json

```diff
@@ -38,6 +38,7 @@
     },
     "macos": {
       "!test_steps": [
+        "swift test",
```

Comment:
minor: you can also consider switching to +test_steps instead of !, so you dont have to repeat swift test part

Reply:
Thanks, I didn't know about +. I have fixed it.